adapters/uform_gen2_qwen_500m/dataloop.json

```diff
@@ -2,13 +2,15 @@
   "name": "uform-gen2-qwen-500m-huggingface-app",
   "description": "UForm-Gen is a small generative vision-language model primarily designed for Image Captioning and Visual Question Answering.",
   "attributes": {
-    "Provider": "Hugging Face",
+    "Provider": "Unum",
```

Comment:
Who is Unum?

Reply:
The provider for this model. I checked with Or, he told me to find the appropriate company that provided each one and add them and ask Liran to add the appropriate logos. Should we only do it if it's a bigger company?